src/colormaps/bin/colorMaptoSLD.py

```diff
@@ -108,34 +104,34 @@ def generateSLD_v1_0_0(gibsColorMaps, layerName, rgbaOrder) :
                     quantity = str(m.group(1))
                     label    = str(m.group(1)) + " - " + str(m.group(2))
                 
-                    print("              <se:ColorMapEntry " + 
+                    print(("              <se:ColorMapEntry " + 
                             "color=\"" + RGBToHex(cmEntry.rgb, "RGB") + "\" " + 
-            		        "quantity=\""+ quantity + "\" " + 
-                		    "label=\""+ label + "\" />")        
-                		    
+                            "quantity=\""+ quantity + "\" " + 
+                            "label=\""+ label + "\" />"))        
+                            
                     quantity = str(m.group(2))
                     label    = str(m.group(2))
                 
-                    print("              <se:ColorMapEntry " + 
+                    print(("              <se:ColorMapEntry " + 
                             "color=\"" + RGBToHex(cmEntry.rgb, "RGB") + "\" " + 
-            		        "quantity=\""+ quantity + "\" " + 
-            		        "label=\""+ label + "\" />")  
-            	else:  
+                            "quantity=\""+ quantity + "\" " + 
+                            "label=\""+ label + "\" />"))  
+                else:  
                     quantity = str(m.group(1))
                     label    = str(m.group(1))
                 
-                    print("              <se:ColorMapEntry " + 
+                    print(("              <se:ColorMapEntry " + 
                             "color=\"" + RGBToHex(cmEntry.rgb, "RGB") + "\" " + 
-            		        "quantity=\""+ quantity + "\" " + 
-            		        "label=\""+ label + "\" />")            
+                            "quantity=\""+ quantity + "\" " + 
+                            "label=\""+ label + "\" />"))            
             else:
                 quantity = str(m.group(1))
                 label    = str(m.group(1)) + " - " + str(m.group(2))#cmEntry.label
                 
-                print("              <se:ColorMapEntry " + 
+                print(("              <se:ColorMapEntry " + 
```

Comment:
Double `((` and `))`.  I see this in a few scripts. Not a problem per se, just an observation